src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml

```diff
@@ -81,3 +86,23 @@
         </div>
     }
 </div>
+<script type="text/javascript" at="Foot">
+    togglePassword = document.querySelector('#togglePassword');
+    password = document.querySelector('#Password');
+
+    togglePassword.addEventListener('click', function (e) {
+        // toggle the type attribute
```

Comment:
cool, lgtm to me.

Only comment is when inlining javascript comments like this they render in the page.

I changed views recently to be razor comments instead, because I felt like we shouldn't be showing our comments to the world, i.e. in the output html.

You might agree / disagree?


Makes sense to me to do this with javascript rather than jquery as this is a view that is often themed, and shouldn't require the theme to use jquery.

Reply:
Makes sense I changed it to Razor comments